src/AppBundle/Controller/Admin/AdController.php

```diff
@@ -37,7 +37,6 @@ public function indexAction(Request $request)
                                  , new EntityDir\User()
                                  , [ 'options' => [ 'roleChoices'        => [EntityDir\User::ROLE_LAY_DEPUTY=>'Lay deputy'],
```

Comment:
or just make the third param a simple array: 

 ['roleChoices'        => [EntityDir\User::ROLE_LAY_DEPUTY=>'Lay deputy'],
'roleNameSetTo'      => EntityDir\User::ROLE_LAY_DEPUTY,
'roleNameEmptyValue' => null,]

Reply:
I'll clean it up further after I write the test, I'll submit for a 2nd review later